core/aws_ddk_core/stages/appflow_ingestion.py

```diff
@@ -84,7 +85,7 @@ def __init__(
             Additional arguments to pass to State Machine creation.
             See: https://awslabs.github.io/aws-ddk/release/latest/api/core/stubs/aws_ddk_core.pipelines.StateMachineStage.html#aws_ddk_core.pipelines.StateMachineStage.build_state_machine # noqa
         """
-        super().__init__(scope, id)
+        super().__init__(scope, id, **kwargs)
```

Comment:
@LeonLuttenberger are there any disadvantages to passing `kwargs` to the parent class in every stage?